scripts/maintenance/update_chromadb_metadata.py

```
@@ -0,0 +1,244 @@
+        record_id = sys.argv[1]
+        
+        # Parse metadata updates from remaining arguments
+        metadata_updates = {}
+        for arg in sys.argv[2:]:
+            if "=" in arg:
+                key, value = arg.split("=", 1)
+                # Try to parse as JSON, otherwise treat as string
+                try:
+                    value = json.loads(value)
+                except:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)